catalog/nsili/catalog-nsili-orb-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,7 +17,7 @@
     <OCD name="NSILI Corba ORB" id="org.codice.alliance.nsili.orb.impl.corbaorb"
          description="NSILI Corba ORB Configuration">
 
-        <AD description="Corba Listen Port."
+        <AD description="Corba Listen Port. NOTE: A restart is required to set the port back to a port number that has already been set."
```

Comment:
What happens if you wait a few minutes between each set? I think that should allow you to then set the same port, without triggering that error. If that's not the case, it could be that object (the ORB?) is not correctly releasing it's resources, since I would think that once its port has changed, it should close and release that socket.